src/Numerical/Spectral/MeanValue.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines function MeanValue.
+
+#pragma once
+
+#include "Domain/Side.hpp"
+#include "Numerical/Spectral/DefiniteIntegral.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+
+/*! \ingroup Functors
+ * \brief Compute the mean value of a grid-function over a manifold.
```

Comment:
Doesn't this just compute the mean value of a function on a grid?

Reply:
No, it performs an integral over a manifold.  It does not take the average of the elements of the DataVector